crates/biome_js_analyze/src/lint/nursery/use_explicit_type.rs

```diff
@@ -362,6 +366,38 @@ fn is_function_used_in_argument_or_array(func: &AnyJsFunction) -> bool {
     )
 }
 
+/// Check if the function is used in some object
+///
+/// # Examples
+///
+/// JS_OBJECT:
+///
+/// interface Behavior {
+///   attribute: string;
+///   func: () => string;
+///   arrowFunc: () => string;
+/// }
+///
+/// function getObjectWithFunction(): Behavior {
+///   return {
+///     attribute: 'value',
+///     func: function myFunc(): string { return "value" },
+///     arrowFunc: () => {},
+///   }
+/// }
```

Comment:
Code example should be in code block.

```suggestion
/// ```ts
/// function getObjectWithFunction(): Behavior {
///   return {
///     attribute: 'value',
///     func: function myFunc(): string { return "value" },
///     arrowFunc: () => {},
///   }
/// }
/// ```
```